### DISCUSSION TITLE
Scoring in case of equal solutions

### DISCUSSION THREAD

**Discussion by dietmarwo** [esa/SpOC3#6] [Challenge 2: Interferometric Mission]
If (hypothetically) there is a challenge so easy, that the absolute optimum can be found in < 10 seconds, how is the ranking handled?
If you are the 11th uploading this optimum, you cannot score at all despite you found the absolute optimum? Maybe it would be better 
to make the challenge slightly harder in this case.

**Comment by darioizzo** [comment]
That is correct. In that case, being the challenge so easy, one could assume the organizers made sure (still in the hypothetical speech) that instance would not influence significantly the final leaderboard.

**Comment by dietmarwo** [comment]
Hallo Dario,
thank you for the clarification. I hope the scenario discussed remains a "what-if" situation, particularly looking forward to varied outcomes in "SpOC 3: Interferometric Mission / Tiny observatory in DRO." I'm somewhat skeptical about the adequacy of an 11x11 grid for fostering innovation.

The discrepancy in scores between the 11th and 1st ranks for what appears to be a straightforward task is intriguing, highlighting a 10-point gap for potentially identical solutions. Given that the overall maximum score is calculated as g = 82.22 = 2*((4/3)^2*10 + (4/3)*10 + 10), the significance of a 10-point difference out of a total of 82 points is debatable.

This might very well be by design, serving as an incentive for early submissions. From my perspective, as the second person to submit, I find this acceptable. However, if there are any plans to adjust this task, it would be best to implement these changes sooner rather than later in the competition.

**Reply by darioizzo** [reply]
You interpreted the rules correctly, well done! 

No plans atm to modify any of the challenges, nor problem instances. We would do this only in case we realised new and unwanted dynamics arising.

**Comment by dietmarwo** [comment]
Last years winner got the same result (3rd team) as 
another 4th team (theEs) , all identical score, see 
https://optimise.esa.int/challenge/spoc-3-interferometric-mission/p/tiny-observatory-in-dro  

> one could assume the organizers made sure (still in the hypothetical speech) that instance would not influence significantly the final leaderboard.

There are many scenarios where this could be relevant, for instance:

1) Imagine theEs winning 2 programmable Cubes tasks (2nd in the easy variant) and additionally obtaining the best scores for all 
three inferometric mission tasks. Uploading as 4th team for the easy inferometric mission means 4 missing points altogether .

2) Imagine HRI winning all 3 torso decomposition tasks and  the easy programmable Cubes task, being second at the medium and
hard variant behind theEs. Means 3.11 missing points altogether. 

In this hypothetical, but not unimaginable scenario it would be decisive that theEs cannot improve at the easy inferometric mission
and HRI would win. 

P.S.
Just today theEs took the lead in inferometric mission (and has it in programmabe cubes) and HRI leads in torso decompositions,
so it is quite unclear what  "the organizers made sure that this instance would not influence significantly the final leaderboard" exactly means. At least as the competition stands now.
